<launch>
	<param name="robot_vel" type="double" value="236.2" />
	<param name="num_robots" type="int" value="3" />
	<param name="range" type="double" value="328.0" />
	<group ns="carter1">
		<rosparam param="starting_stations">['WS14', 'WS10', 'WS7', 'WS9', 'WS2'] </rosparam>
		<rosparam param="starting_segments">[['v', 'W', 'X', 'Q', 'r'], ['r', 'Q', 'O', 'o'], ['v', 'W', 'X', 'Q', 'r', 'q'], ['o', 'O', 'J', 'G', 'F', 'j']] </rosparam>
		<param name = "initial_station" type="string" value="WS11"/>
		<param name="map_yaml_path" value ="home/russell/thesis_ws/src/thesis_sim/ddzoning/assets/carter_LEfloor_navigation.yaml" />
        	<param name="iteration_count" type="int" value="50000" />
        	<param name="goal_generator_type" type="str" value="GoalReader" />
        	<param name="action_server_name" type="str" value="move_base" />
        	<param name="obstacle_search_distance_in_meters" type="double" value="0.2" />
        	<param name="goal_text_file_path" value="home/russell/thesis_ws/src/thesis_sim/ddzoning/assets/goals1.txt" />
        	<rosparam param="initial_pose">[22.0, 3.0, 0, 0, 0, 1, 0] </rosparam>
        	<node name="robot_node" pkg="ddzoning" type="robot.py" output="screen"/>
        	<node name="set_navigation_goal" pkg="ddzoning" type="set_goal.py" output="screen"/>
	</group>
	<group ns="carter2">
		<rosparam param="starting_stations">['WS1', 'WS4', 'WS5', 'WS3'] </rosparam>
		<rosparam param="starting_segments">[['i', 'B', 'C', 'D', 'l'], ['i', 'A', 'E', 'I', 'H', 'm'], ['m', 'H', 'I', 'E', 'k']]  </rosparam>
		<param name = "initial_station" type="string" value="WS1"/>
	        <param name="map_yaml_path" value="home/russell/thesis_ws/src/thesis_sim/ddzoning/assets/carter_LEfloor_navigation.yaml" />
		<param name="iteration_count" type="int" value="50000" />
		<param name="goal_generator_type" type="str" value="GoalReader" />
		<param name="action_server_name" type="str" value="move_base" />
		<param name="obstacle_search_distance_in_meters" type="double" value="0.2" />
		<param name="goal_text_file_path" value="home/russell/thesis_ws/src/thesis_sim/ddzoning/assets/goals2.txt" />
		<rosparam param="initial_pose">[9.0, 84.0, 0, 0, 0, 1, 0] </rosparam>
		<node name="robot_node" pkg="ddzoning" type="robot.py" output="screen"/>
		<node name="set_navigation_goal" pkg="ddzoning" type="set_goal.py" output="screen"/>
	</group>
	<group ns="carter3">
		<rosparam param="starting_stations">['WS17', 'WS16', 'WS12', 'WS13', 'WS11', 'WS8', 'WS6', 'WS18', 'WS15'] </rosparam>
		<rosparam param="starting_segments">[['y', 'a', 'Z', 'x'], ['x', 'Z', 't'], ['u', 'a', 'Z', 'Y', 's'], ['u', 'V', 'U', 't'], ['t', 'U', 'T', 'S', 'R', 'p'], ['p', 'L', 'M', 'N', 'n'], ['s', 'Y', 'Z', 'a', 'b', 'z'], ['z', 'b', 'c', 'w']] </rosparam>
		<param name = "initial_station" type="string" value="WS18"/>
	        <param name="map_yaml_path" value="home/russell/thesis_ws/src/thesis_sim/ddzoning/assets/carter_LEfloor_navigation.yaml" />
		<param name="iteration_count" type="int" value="50000" />
		<param name="goal_generator_type" type="str" value="GoalReader" />
		<param name="action_server_name" type="str" value="move_base" />
		<param name="obstacle_search_distance_in_meters" type="double" value="0.2" />
		<param name="goal_text_file_path" value="home/russell/thesis_ws/src/thesis_sim/nvidia_sim/assets/goals3.txt" />
		<rosparam param="initial_pose">[70.0, 3.0, 0, 0, 0, 1, 0] </rosparam>
		<node name="robot_node" pkg="ddzoning" type="robot.py" output="screen"/>
		<node name="set_navigation_goal" pkg="ddzoning" type="set_goal.py" output="screen"/>
	</group>
	<node name="map_node" pkg="ddzoning" type="map.py" output="screen"/>
</launch>
